extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts

```diff
@@ -94,8 +93,8 @@ export class EncryptedMsgMailFormatter extends BaseMailFormatter {
 
   private formatSendablePwdMsgs = async (newMsg: NewMsgData, pubkeys: PubkeyResult[], signingKey?: ParsedKeyInfo) => {
     // password-protected message, temporarily uploaded (already encrypted) to:
-    //    - flowcrypt.com/api (consumers and customers without on-prem setup), or
-    //    - FlowCrypt Enterprise Server (enterprise customers with on-prem setup)
+    //    - flowcrypt.com/shared-tenant-fes (consumers and customers without on-prem setup), or
+    //    - FlowCrypt External Service (enterprise customers with on-prem setup)
```

Comment:
`FlowCrypt External Service` here can be replaced with `fes.customer-domain.com`